railties/lib/rails/commands/credentials/credentials_command.rb

```diff
@@ -55,10 +55,6 @@ def key_path
           options[:environment] ? "config/credentials/#{options[:environment]}.key" : "config/master.key"
         end
 
-        def env_key
-          options[:environment] ? "RAILS_#{options[:environment].upcase}_KEY" : "RAILS_MASTER_KEY"
-        end
-
 
```

Comment:
This double blank lines are intended?

Reply:
Yes, similar to what is in https://github.com/rails/rails/blob/b4b70ef2bfd922723aa02578d3feba887f5249dc/railties/lib/rails/commands/credentials/credentials_command.rb#L73-L74
I guess it was called "separation of concerns" in some commit, but I can't find it :-)